api/envoy/config/filter/network/http_connection_manager/v2/http_connection_manager.proto

```diff
@@ -328,6 +328,12 @@ message HttpConnectionManager {
     // Defaults to false.
     bool cert = 3;
 
+    // Whether to forward the entire client cert chain (including the leaf cert) in URL encoded PEM
```

Comment:
Add comment `// [#comment:next free field: 7]` to the top of `SetCurrentClientCertDetails`, or put this new field at the bottom?